my-use-case/build_image.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+image_name=gcr.io/my-org/my-image
+image_tag=latest
+full_image_name=${image_name}:${image_tag}
+
+cd "$(dirname "$0")"
+docker build -t "${full_image_name}" .
+docker push "$full_image_name"
+
+# Output the strict image name, which contains the sha256 image digest
+docker inspect --format="{{index .RepoDigests 0}}" "${full_image_name}"
```

Comment:
Ideally - capture the full image digest / strict name and use this in the pipeline component definition at compilation

Reply:
would like to change this to cloudbuild.yaml as well. open for ideas on how to keep the python package version and docker build in sync.